tools/PowerShell/New-CreateSearchIndexerDefaultValueSets.ps1

```diff
@@ -0,0 +1,61 @@
+<#
+.SYNOPSIS
+This script downloads the spec definitions and creates a mapping file of a FHIR Resource property, to its specified CodeSystem,
+e.g. "DocumentReference.relatesTo.code":  "http://hl7.org/fhir/document-relationship-type"
+#>
```

Comment:
Could you add a note somewhere to remind people to run this script when the FHIR version is updated? It looks like this will need to be rerun whenever the FHIR spec is updated.

Reply:
I agree, there are now a few steps in updating a FHIR version. I'll probably address this as another story (i.e. a how-to for updating a FHIR version document) that also includes things like #901

Reply:
Also adding #1089